build.gradle

```diff
@@ -250,6 +255,7 @@ project('client') {
         compile project(':shared:authplugin')
         compile project(':shared:protocol')
         compile project(":shared:controller-api")
+        compile project(":shared:security")
```

Comment:
This makes the client depend on `jaxb-api` and `jaxb-runtime`. Are these needed for this change to work? If not please find a way to reconfigure things so the client isn't pulling these in.

Reply:
These two jaxb dependencies seem to not be needed anymore, so I've removed these now. As for the client depending on the `:shared:security` project, we have introduced a new class `AccessOperation` that belongs to this shared project, as it is used in both the client and the server.